MAUI/TabView/Header-Display-Mode.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Header Display Mode in Maui Tabbed View control | Syncfusion
+description: Learn here all about Header Display Mode support in Syncfusion Maui Tabbed View (SfTabView) control and more.
+platform: maui
+control: TabView
+documentation: ug
+---
+
+# Header Display Mode in Maui Tabbed View (SfTabView)
+
+By default, the tab view control displays the title of each tab item. It can be changed to any of the following types:
+
+* Default
+* Image
+* Text
+
+![Image for HeaderDisplayMode](images/Header-Display-Mode.png)
+
+The tab view can be changed by setting the `HeaderDisplayMode` property of [`SfTabView`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.TabView.html).
```

Comment:
use "Tab View" instead of "tab view"

Reply:
Changed the text from tab view to Tab View.